app/templates/profile/profile_main.html

```diff
@@ -1,11 +1,9 @@
-{% extends "layout.html" %}
-
-{% block content %}
+{% extends "twoDiv.html" %}
```

Comment:
Rather than extending `layout.html` directly, the developer would extend our `twoDiv.html` template. So, if another dev independently wanted to utilize the two div structure, they would be consistent and wouldn't be recreating the structure in multiple places.

Reply:
Probably don't need to do this.